release.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+set -euo pipefail
+
+NEW_VERSION="$(date +%Y.%m.%d)"
+OLD_VERSION="$(git tag --list | tail -n 1)"
```

Comment:
We can run fetch -p --all before just to make sure we get updates from upstream and then make sure that other folk release tag will be fetched before listing the tags.